src/ci/docker/host-x86_64/dist-loongarch64-linux/loongarch64-unknown-linux-gnu.defconfig

```diff
@@ -0,0 +1,19 @@
+CT_CONFIG_VERSION="4"
+CT_EXPERIMENTAL=y
+CT_PREFIX_DIR="/x-tools/${CT_TARGET}"
+CT_LOG_ALL=y
```

Comment:
We don't really need logging, but I guess it doesn't hurt either way.

Please do add the mirror config like the other targets though:

```
CT_USE_MIRROR=y
CT_MIRROR_BASE_URL="https://ci-mirrors.rust-lang.org/rustc"
```

(We don't have everything mirrored, but AIUI this will let our mirror take precedence when we do add things.)